tests/baseSetup.go

```diff
@@ -20,40 +20,44 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/rook/rook/pkg/test/enums"
-	"github.com/rook/rook/pkg/test/manager"
-	"github.com/rook/rook/pkg/test/objects"
+	"github.com/rook/rook/tests/framework/enums"
+	"github.com/rook/rook/tests/framework/installer"
+	"github.com/rook/rook/tests/framework/objects"
 )
 
 var (
 	Env      objects.EnvironmentManifest
+	rook     *installer.InstallHelper
 	Platform enums.RookPlatformType
 )
 
 //One init function per package - initializes Rook infra and installs rook(if needed based on flags)
 func init() {
 	Env = objects.NewManifest()
 	var err error
-
-	rookPlatform, err := enums.GetRookPlatFormTypeFromString(Env.Platform)
+	platform, err := enums.GetRookPlatFormTypeFromString(Env.Platform)
```

Comment:
is this still needed? I thought we're going to remove the platform abstraction and go with specific test suites for each platform?